### DISCUSSION TITLE
latest beta 8.0.10 beta not working with Firefox 106.0

### DISCUSSION THREAD

**Discussion by Raul1198** [subhra74/xdm#914] [Q&A]
i installed the extension via the in app Firefox link and when i download anything it downloads with Firefox inbuilt download even the extension is enabled (blue XDM icon is shown in Firefox) extension and the app is running

**Comment by prajwalkarna** [comment]
same case with me

**Comment by subhra74** [comment]
Please install latest version from https://github.com/subhra74/xdm-experimental-binaries/releases/tag/8.0.11-beta.
The browser extension must be installed from XDM browser monitoring screen. Instructions to install the extension is provided there as well.

**Comment by Raul1198** [comment]
Still same on FF 106.0.1 Win11 22H2
